src/RNCamera.js

```diff
@@ -342,159 +342,258 @@ const mapValues = (input, mapper) => {
   return result;
 };
 
-export default class Camera extends React.Component<PropsType, StateType> {
-  static Constants = {
-    Type: CameraManager.Type,
-    FlashMode: CameraManager.FlashMode,
-    AutoFocus: CameraManager.AutoFocus,
-    WhiteBalance: CameraManager.WhiteBalance,
-    VideoQuality: CameraManager.VideoQuality,
-    VideoCodec: CameraManager.VideoCodec,
-    BarCodeType: CameraManager.BarCodeType,
-    GoogleVisionBarcodeDetection: CameraManager.GoogleVisionBarcodeDetection,
-    FaceDetection: CameraManager.FaceDetection,
-    CameraStatus,
-    RecordAudioPermissionStatus: RecordAudioPermissionStatusEnum,
-    VideoStabilization: CameraManager.VideoStabilization,
-    Orientation: {
-      auto: 'auto',
-      landscapeLeft: 'landscapeLeft',
-      landscapeRight: 'landscapeRight',
-      portrait: 'portrait',
-      portraitUpsideDown: 'portraitUpsideDown',
-    },
-  };
+export const Constants = {
```

Comment:
we need to update the [readme][1] and users would need to update their application to `import { Constants } from 'react-native-camera'` and use `Constants.VideoQuality.2160p`. The [basic example][3] needs change. 

Would be better exporting `Constants` as part of the `RNCamera.Constants` namespace to avoid changes from the previous api, probably it can be accomplished by including `Constants` inside the `Camera` function.

[1]: https://github.com/react-native-community/react-native-camera/blob/master/docs/RNCamera.md#defaultvideoquality
[3]: https://github.com/react-native-community/react-native-camera/blob/55ff9809acf803bcbdf93a1a76b763d0d97ce512/examples/basic/App.js#L65